processing/tests/test_datasets/metadata.json

```diff
@@ -0,0 +1 @@
+{"decipher_version": "0.1.29"}
```

Comment:
If we update as written above, remember to include updating the version tag here.

Reply:
Temporarily set the version tag to 0.1.29 to make sure it's working with the rest in this branch. This is more up to date on [CAN-358/add/filters-on-landing-page](https://github.com/Simula-Consulting/decipher/tree/CAN-358/add/filters-on-landing-page)

Reply:
It now crashes if you try to serve without having pip installed pyarrow manually, but whatever, it's fixed on the new branch